src/runtime-rs/crates/runtimes/src/manager.rs

```diff
@@ -311,6 +325,13 @@ impl RuntimeHandlerManager {
             }
             Request::UpdateContainer(req) => {
                 cm.update_container(req).await.context("update container")?;
+
+                // update the sandbox's resource when a container's spec is updated
+                instance
```

Comment:
@Ji-Xinyou Did you update the `oci spec` when the update has been called? It seems you are still using the original one?